tests/test_api.py

```diff
@@ -0,0 +1,332 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""System tests for Electricity Trading API."""
+import asyncio
+import logging
+import os
+from datetime import datetime, timedelta, timezone
+from decimal import Decimal
+from typing import Any
+
+import grpc
+import pytest
+
+from frequenz.client.electricity_trading import (
+    Client,
+    Currency,
+    DeliveryArea,
+    DeliveryPeriod,
+    Energy,
+    EnergyMarketCodeType,
+    MarketSide,
+    OrderDetail,
+    OrderState,
+    OrderType,
+    Price,
+)
+
+# Setup logging configuration
+logging.basicConfig(
+    level=logging.DEBUG, format="%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+)
+_logger = logging.getLogger(__name__)
+
+GRIDPOOL_ID = 6
+API_KEY = os.getenv("API_KEY", None)
+if not API_KEY:
+    raise ValueError("API_KEY is not set")
+
+
+@pytest.fixture
+async def set_up() -> dict[str, Any]:
+    """Set up the test suite."""
+    client = Client(
+        server_url="grpc://electricity-trading-testing.api.frequenz.com:443?ssl=true",
+        auth_key=API_KEY,
+    )
+
+    delivery_area = DeliveryArea(
+        code="10YDE-EON------1", code_type=EnergyMarketCodeType.EUROPE_EIC
+    )
+    # Setting delivery start to the next day 12:00
```

Comment:
The comment states that its creating a delivery start at 12:00 the next day, but to my understanding `delivery_start = (datetime.now(timezone.utc) + timedelta(hours=3)).replace(
        minute=0, second=0, microsecond=0
    )` uses the current hour + 3 hours and not 12:00 on the next day. Or am I misunderstanding anything?

Reply:
Probably the comment is outdated?

Reply:
Indeed good call! Fixed it now